docs/Setup/WSL-Guide.md

```diff
@@ -0,0 +1,61 @@
+This doc was written on `Windows 10 Version 2004 (OS Build 19041.388)` using WSL 2 running Ubuntu 20.04.
+
+Powershell:
+
+```
+PS C:\Users\justin> wsl --list --verbose
+  NAME                   STATE           VERSION
+* Ubuntu-20.04           Running         2
+  docker-desktop         Running         2
+  docker-desktop-data    Running         2
+```
+
+### Install Docker on windows
+
+Make sure you set it up with WSL.
+
+https://docs.docker.com/docker-for-windows/
+
+https://docs.docker.com/docker-for-windows/wsl/
+
+After its running (on Windows), it should be added to your WSL path.
+
+### Install Vagrant on WSL
+
+You probably need to install Virtualbox too, as it says in the article.
+
+https://linuxize.com/post/how-to-install-vagrant-on-ubuntu-20-04/
```

Comment:
I'd link to https://www.vagrantup.com/intro/getting-started/install instead

Reply:
I linked to that article as it's a way of doing it in the terminal directly; Vagrant's official releases page only provides links to download `zip`s, which devs less experienced with bash may not know how to install onto WSL (eg: I don't)

Reply:
/shrug okay